epoxy-composeinterop-maverickssample/build.gradle

```diff
@@ -0,0 +1,65 @@
+plugins {
+  id 'com.android.application'
+  id 'kotlin-android'
+}
```

Comment:
don't include the mvrx changes in this PR to make it reviewable. either change the base branch or remove all of these files